### PR TITLE
add comments list to article page

### PR DESCRIPTION
**Pull request by katjakrempel**
completed task 6 - view list of comments associated with an article

### CODE REVIEW

File: utils/api.js [hunk 1]
```diff
@@ -4,10 +4,14 @@ const api = axios.create({
...
 
-export function fetchArticleById(article_id) {
+export function getArticleById(article_id) {
     return api.get(`/articles/${article_id}`).then((response) => response.data.article)
-}
\ No newline at end of file
+};
+
+export function getCommentsByArticleId(article_id) {
+return api.get(`/articles/${article_id}/comments`).then((response)=> response.data.comments)
```
Comment (MRombi) [inline utils/api.js:16]: We should not need a .then here 🙂

File: src/components/SingleArticle.jsx [hunk 2]
```diff
@@ -1,32 +1,47 @@
...
+    getArticleById(article_id).then((response) => {
       setArticle(response);
       setIsLoading(false);
     });
+    getCommentsByArticleId(article_id).then((response) => {
+      setComments(response);
+    });
   }, [article_id]);
 
   if (isLoading) return <p>Loading...</p>;
```
Comment (MRombi) [inline src/components/SingleArticle.jsx:23]: We should have a separate loading screen for the articles and the comments so the user does not wait for all the comments to render to read the article 🙂

**Review by MRombi — COMMENTED** [review]
Good job overall, take into account the feedback I left before proceeding 🙂

File: utils/api.js (hunk 1 above)
Comment (katjakrempel) [reply utils/api.js:16]: fixed in #5

File: src/components/SingleArticle.jsx (hunk 2 above)
Comment (katjakrempel) [reply src/components/SingleArticle.jsx:23]: fixed in #5

**Pull request merged**